stable/fluentd-shipper/values.yaml

```diff
@@ -0,0 +1,19 @@
+image: "anchorfree/fluentd:v3.0"
+env:
+- name: FLUENTD_FORWARD_HOST
+  value: "fluentd.elasticsearch.afdevops.com"
```

Comment:
That address is obsolete, could you please consult with @pavdmyt about new dns name

Reply:
Please use `input.efk.afdevops.com` instead.